inc/namespace.php

```diff
@@ -3,11 +3,31 @@
 namespace Altis\Local_Server;
 
 use function Altis\get_config;
+use function Altis\get_environment_type;
 
 /**
  * Configure environment for local server.
  */
 function bootstrap() {
+	if ( get_environment_type() === 'local' ) {
+		add_filter( 'admin_menu', function() {
```

Comment:
I know it's not all that consistent where we use anonymous functions in this file but I think it'll keep things tidier from now on to just have the filter call here.

Reply:
Sorry to clarify this I meant:

- move the anonymous function out to the bottom of the file
- call `add_filter( 'admin_menu', __NAMESPACE__ . '\\tools_menu' );` or whatever you name the function